neuro-gui.py

```diff
@@ -156,23 +182,35 @@ def _setMainUi(self):
         self.setCentralWidget(self.widget)
 
     def log_subprocess_output(self, pipe):
```

Comment:
This function was only for the test and it is probably not needed. I used it to get the output from the image generator into the debug window. But it didn't work